### PR TITLE
Add basic quantized operators

### PR DESCRIPTION
**Pull request by hjhee**
- [x] _fused_moving_avg_obs_fq_helper
- [x] _fake_quantize_learnable_per_channel_affine
- [x] _fake_quantize_learnable_per_channel_affine_backward
- [x] _fake_quantize_learnable_per_tensor_affine
- [x] _fake_quantize_learnable_per_tensor_affine_backward
- [x] _fake_quantize_per_tensor_affine_cachemask_tensor_qparams
- [x] _empty_affine_quantized
- [x] _empty_per_channel_affine_quantized
- [x] quantize_per_channel
- [x] quantize_per_tensor
- [x] quantize_per_tensor_dynamic
- [x] quantize_per_tensor.tensor_qparams
- [x] dequantize.self
- [x] _make_per_tensor_quantized_tensor
- [x] _make_per_channel_quantized_tensor
- [x] fake_quantize_per_channel_affine_cachemask
- [x] fake_quantize_per_tensor_affine_cachemask

### CODE REVIEW

**Comment by hjhee** [comment]
- <div><b>PASSED</b></div>
<div>test_workflow_ops_xpu.py::<b>TestFusedObsFakeQuant</b>XPU::test_fused_backward_op_fake_quant_off_xpu</div>
<div>test_workflow_ops_xpu.py::<b>TestFusedObsFakeQuant</b>XPU::test_fused_obs_fake_quant_backward_op_xpu</div>
<div>test_workflow_ops_xpu.py::<b>TestFusedObsFakeQuant</b>XPU::test_fused_obs_fake_quant_moving_avg_xpu</div>
<div>test_workflow_ops_xpu.py::<b>TestFakeQuantizeOps</b>XPU::test_backward_per_channel_cachemask_cpu_xpu</div>
<div>test_workflow_ops_xpu.py::<b>TestFakeQuantizeOps</b>XPU::test_backward_per_channel_cachemask_cuda_xpu</div>
<div>test_workflow_ops_xpu.py::<b>TestFakeQuantizeOps</b>XPU::test_backward_per_tensor_cachemask_cpu_xpu</div>
<div>test_workflow_ops_xpu.py::<b>TestFakeQuantizeOps</b>XPU::test_backward_per_tensor_cachemask_cuda_xpu</div>
<div>test_workflow_ops_xpu.py::<b>TestFakeQuantizeOps</b>XPU::test_fake_quant_control_xpu</div>
<div>test_workflow_ops_xpu.py::<b>TestFakeQuantizeOps</b>XPU::test_fake_quant_per_channel_qparam_range_xpu</div>
<div>test_workflow_ops_xpu.py::<b>TestFakeQuantizeOps</b>XPU::test_fake_quant_preserves_qparam_shapes_for_activations_xpu</div>
<div>test_workflow_ops_xpu.py::<b>TestFakeQuantizeOps</b>XPU::test_fake_quantize_per_channel_affine_scale_dtypes_xpu</div>
<div>test_workflow_ops_xpu.py::<b>TestFakeQuantizeOps</b>XPU::test_fixed_qparams_fq_module_xpu</div>
<div>test_workflow_ops_xpu.py::<b>TestFakeQuantizeOps</b>XPU::test_forward_backward_per_tensor_with_amp_xpu</div>
<div>test_workflow_ops_xpu.py::<b>TestFakeQuantizeOps</b>XPU::test_forward_per_channel_cachemask_cpu_xpu</div>
<div>test_workflow_ops_xpu.py::<b>TestFakeQuantizeOps</b>XPU::test_forward_per_channel_cachemask_cuda_xpu</div>
<div>test_workflow_ops_xpu.py::<b>TestFakeQuantizeOps</b>XPU::test_forward_per_channel_half_precision_numerics_xpu</div>
<div>test_workflow_ops_xpu.py::<b>TestFakeQuantizeOps</b>XPU::test_forward_per_channel_xpu</div>
<div>test_workflow_ops_xpu.py::<b>TestFakeQuantizeOps</b>XPU::test_forward_per_tensor_cachemask_cpu_xpu</div>
<div>test_workflow_ops_xpu.py::<b>TestFakeQuantizeOps</b>XPU::test_forward_per_tensor_cachemask_cuda_xpu</div>
<div>test_workflow_ops_xpu.py::<b>TestFakeQuantizeOps</b>XPU::test_forward_per_tensor_half_precision_numerics_xpu</div>
<div>test_workflow_ops_xpu.py::<b>TestFakeQuantizeOps</b>XPU::test_forward_per_tensor_xpu</div>
<div>test_workflow_ops_xpu.py::<b>TestFakeQuantizeOps</b>XPU::test_fq_module_per_tensor_xpu</div>
<div>test_workflow_ops_xpu.py::<b>TestFakeQuantizeOps</b>XPU::test_fq_serializable_per_tensor_xpu</div>
<div>test_workflow_ops_xpu.py::<b>TestFakeQuantizeOps</b>XPU::test_learnable_backward_per_channel_cuda_xpu</div>
<div>test_workflow_ops_xpu.py::<b>TestFakeQuantizeOps</b>XPU::test_learnable_backward_per_tensor_cpu_xpu</div>
<div>test_workflow_ops_xpu.py::<b>TestFakeQuantizeOps</b>XPU::test_learnable_backward_per_tensor_cuda_xpu</div>
<div>test_workflow_ops_xpu.py::<b>TestFakeQuantizeOps</b>XPU::test_learnable_forward_per_channel_cpu_xpu</div>
<div>test_workflow_ops_xpu.py::<b>TestFakeQuantizeOps</b>XPU::test_learnable_forward_per_channel_cuda_xpu</div>
<div>test_workflow_ops_xpu.py::<b>TestFakeQuantizeOps</b>XPU::test_learnable_forward_per_tensor_cuda_xpu</div>
<div>test_workflow_ops_xpu.py::<b>TestFakeQuantizeOps</b>XPU::test_numerical_consistency_per_channel_xpu</div>
<div>test_workflow_ops_xpu.py::<b>TestFakeQuantizeOps</b>XPU::test_numerical_consistency_per_tensor_xpu</div>
<div>test_workflow_module_xpu.py::<b>TestFakeQuantize</b>XPU::test_fq_module_per_channel_xpu</div>
<div>test_workflow_module_xpu.py::<b>TestFakeQuantize</b>XPU::test_fq_serializable_per_channel_xpu</div>
<div>test_workflow_module_xpu.py::<b>TestFakeQuantize</b>XPU::test_quant_min_max_override_xpu</div>
<div>test_workflow_module_xpu.py::<b>TestFusedObsFakeQuant</b>ModuleXPU::test_compare_fused_obs_fq_oss_module_xpu</div>
<div>test_workflow_module_xpu.py::<b>TestFusedObsFakeQuant</b>ModuleXPU::test_default_fused_qat_config_xpu</div>
<div>test_workflow_module_xpu.py::<b>TestFusedObsFakeQuant</b>ModuleXPU::test_embedding_bag_qat_config_xpu</div>
<div>test_workflow_module_xpu.py::<b>TestFusedObsFakeQuant</b>ModuleXPU::test_embedding_qat_config_xpu</div>
<div>test_workflow_module_xpu.py::<b>TestFusedObsFakeQuant</b>ModuleXPU::test_fused_mod_per_channel_xpu</div>
<div>test_workflow_module_xpu.py::<b>TestFusedObsFakeQuant</b>ModuleXPU::test_fused_mod_reduce_range_xpu</div>
<div>test_workflow_module_xpu.py::<b>TestFusedObsFakeQuant</b>ModuleXPU::test_fused_obs_fq_module_xpu</div>
<div>test_workflow_module_xpu.py::<b>TestFusedObsFakeQuant</b>ModuleXPU::test_fused_obs_fq_moving_avg_module_xpu</div>
<div>test_quantized_tensor_xpu.py::<b>TestQuantizedTensor</b>XPU::test_bfp16_quantize_xpu</div>
<div>test_quantized_tensor_xpu.py::<b>TestQuantizedTensor</b>XPU::test_choose_qparams_optimized_xpu</div>
<div>test_quantized_tensor_xpu.py::<b>TestQuantizedTensor</b>XPU::test_clone_xpu</div>
<div>test_quantized_tensor_xpu.py::<b>TestQuantizedTensor</b>XPU::test_decomposed_choose_qparams_per_token_asymmetric_backward_xpu</div>
<div>test_quantized_tensor_xpu.py::<b>TestQuantizedTensor</b>XPU::test_decomposed_dequantize_per_channel_xpu</div>
<div>test_quantized_tensor_xpu.py::<b>TestQuantizedTensor</b>XPU::test_decomposed_dequantize_per_tensor_xpu</div>
<div>test_quantized_tensor_xpu.py::<b>TestQuantizedTensor</b>XPU::test_decomposed_dynamic_quant_pattern_xpu</div>
<div>test_quantized_tensor_xpu.py::<b>TestQuantizedTensor</b>XPU::test_decomposed_quantize_per_channel_bfloat16_input_xpu</div>
<div>test_quantized_tensor_xpu.py::<b>TestQuantizedTensor</b>XPU::test_decomposed_quantize_per_channel_group_xpu</div>
<div>test_quantized_tensor_xpu.py::<b>TestQuantizedTensor</b>XPU::test_decomposed_quantize_per_channel_xpu</div>
<div>test_quantized_tensor_xpu.py::<b>TestQuantizedTensor</b>XPU::test_decomposed_quantize_per_tensor_bfloat16_input_xpu</div>
<div>test_quantized_tensor_xpu.py::<b>TestQuantizedTensor</b>XPU::test_decomposed_quantize_per_tensor_xpu</div>
<div>test_quantized_tensor_xpu.py::<b>TestQuantizedTensor</b>XPU::test_decomposed_quantize_per_token_xpu</div>
<div>test_quantized_tensor_xpu.py::<b>TestQuantizedTensor</b>XPU::test_dequantize_fp16_cpu_xpu</div>
<div>test_quantized_tensor_xpu.py::<b>TestQuantizedTensor</b>XPU::test_dequantize_fp16_cuda_xpu</div>
<div>test_quantized_tensor_xpu.py::<b>TestQuantizedTensor</b>XPU::test_fp16_saturate_op_xpu</div>
<div>test_quantized_tensor_xpu.py::<b>TestQuantizedTensor</b>XPU::test_jit_serialization_xpu</div>
<div>test_quantized_tensor_xpu.py::<b>TestQuantizedTensor</b>XPU::test_per_channel_qtensor_creation_cpu_xpu</div>
<div>test_quantized_tensor_xpu.py::<b>TestQuantizedTensor</b>XPU::test_per_channel_qtensor_to_memory_format_xpu</div>
<div>test_quantized_tensor_xpu.py::<b>TestQuantizedTensor</b>XPU::test_per_tensor_qtensor_to_memory_format_xpu</div>
<div>test_quantized_tensor_xpu.py::<b>TestQuantizedTensor</b>XPU::test_pickle_checkpoint_qtensor_xpu</div>
<div>test_quantized_tensor_xpu.py::<b>TestQuantizedTensor</b>XPU::test_qscheme_pickle_xpu</div>
<div>test_quantized_tensor_xpu.py::<b>TestQuantizedTensor</b>XPU::test_qtensor_channel_float_assignment_xpu</div>
<div>test_quantized_tensor_xpu.py::<b>TestQuantizedTensor</b>XPU::test_qtensor_copy_xpu</div>
<div>test_quantized_tensor_xpu.py::<b>TestQuantizedTensor</b>XPU::test_qtensor_cpu_xpu</div>
<div>test_quantized_tensor_xpu.py::<b>TestQuantizedTensor</b>XPU::test_qtensor_creation_xpu</div>
<div>test_quantized_tensor_xpu.py::<b>TestQuantizedTensor</b>XPU::test_qtensor_dtypes_xpu</div>
<div>test_quantized_tensor_xpu.py::<b>TestQuantizedTensor</b>XPU::test_qtensor_equal_xpu</div>
<div>test_quantized_tensor_xpu.py::<b>TestQuantizedTensor</b>XPU::test_qtensor_fill_per_channel_nhwc_xpu</div>
<div>test_quantized_tensor_xpu.py::<b>TestQuantizedTensor</b>XPU::test_qtensor_fill_per_channel_xpu</div>
<div>test_quantized_tensor_xpu.py::<b>TestQuantizedTensor</b>XPU::test_qtensor_fill_per_tensor_nhwc_xpu</div>
<div>test_quantized_tensor_xpu.py::<b>TestQuantizedTensor</b>XPU::test_qtensor_fill_per_tensor_xpu</div>
<div>test_quantized_tensor_xpu.py::<b>TestQuantizedTensor</b>XPU::test_qtensor_float_assignment_xpu</div>
<div>test_quantized_tensor_xpu.py::<b>TestQuantizedTensor</b>XPU::test_qtensor_index_put_cpu_xpu</div>
<div>test_quantized_tensor_xpu.py::<b>TestQuantizedTensor</b>XPU::test_qtensor_index_select_cpu_xpu</div>
<div>test_quantized_tensor_xpu.py::<b>TestQuantizedTensor</b>XPU::test_qtensor_int_repr_xpu</div>
<div>test_quantized_tensor_xpu.py::<b>TestQuantizedTensor</b>XPU::test_qtensor_legacy_new_failure_xpu</div>
<div>test_quantized_tensor_xpu.py::<b>TestQuantizedTensor</b>XPU::test_qtensor_load_save_xpu</div>
<div>test_quantized_tensor_xpu.py::<b>TestQuantizedTensor</b>XPU::test_qtensor_masked_fill_cpu_xpu</div>
<div>test_quantized_tensor_xpu.py::<b>TestQuantizedTensor</b>XPU::test_qtensor_per_channel_load_save_xpu</div>
<div>test_quantized_tensor_xpu.py::<b>TestQuantizedTensor</b>XPU::test_qtensor_per_channel_permute_xpu</div>
<div>test_quantized_tensor_xpu.py::<b>TestQuantizedTensor</b>XPU::test_qtensor_permute_xpu</div>
<div>test_quantized_tensor_xpu.py::<b>TestQuantizedTensor</b>XPU::test_qtensor_quant_dequant_xpu</div>
<div>test_quantized_tensor_xpu.py::<b>TestQuantizedTensor</b>XPU::test_qtensor_quantize_per_channel_xpu</div>
<div>test_quantized_tensor_xpu.py::<b>TestQuantizedTensor</b>XPU::test_qtensor_reshape_xpu</div>
<div>test_quantized_tensor_xpu.py::<b>TestQuantizedTensor</b>XPU::test_qtensor_resize_xpu</div>
<div>test_quantized_tensor_xpu.py::<b>TestQuantizedTensor</b>XPU::test_qtensor_sub_byte_aligned_cols_xpu</div>
<div>test_quantized_tensor_xpu.py::<b>TestQuantizedTensor</b>XPU::test_qtensor_sub_byte_not_aligned_cols_xpu</div>
<div>test_quantized_tensor_xpu.py::<b>TestQuantizedTensor</b>XPU::test_qtensor_unsqueeze_xpu</div>
<div>test_quantized_tensor_xpu.py::<b>TestQuantizedTensor</b>XPU::test_qtensor_view_xpu</div>
<div>test_quantized_tensor_xpu.py::<b>TestQuantizedTensor</b>XPU::test_quant_pin_memory_xpu</div>
<div>test_quantized_tensor_xpu.py::<b>TestQuantizedTensor</b>XPU::test_quantize_per_channel_float_qparams_xpu</div>
<div>test_quantized_tensor_xpu.py::<b>TestQuantizedTensor</b>XPU::test_quantize_per_channel_sub_byte_xpu</div>
<div>test_quantized_tensor_xpu.py::<b>TestQuantizedTensor</b>XPU::test_repeat_xpu</div>
<div>test_quantized_tensor_xpu.py::<b>TestQuantizedTensor</b>XPU::test_torch_qtensor_deepcopy_xpu</div>


- <div><b>SKIPPED</b></div>
<div>test_workflow_ops_xpu.py::<b>TestFakeQuantizeOps</b>XPU::test_backward_per_channel_xpu</div>
<div>test_workflow_ops_xpu.py::<b>TestFakeQuantizeOps</b>XPU::test_backward_per_tensor_xpu</div>
<div>test_workflow_ops_xpu.py::<b>TestFakeQuantizeOps</b>XPU::test_learnable_backward_per_channel_cpu_xpu</div>
<div>test_workflow_ops_xpu.py::<b>TestFakeQuantizeOps</b>XPU::test_learnable_forward_per_tensor_cpu_xpu</div>
<div>test_quantized_tensor_xpu.py::<b>TestQuantizedTensor</b>XPU::test_choose_qparams_xpu</div>

File: test/xpu/quantization/core/test_quantized_tensor_xpu.py [hunk 1]
```diff
@@ -0,0 +1,184 @@
+# Owner(s): ["module: intel"]
```
Comment (ZhiweiYan-96) [inline test/xpu/quantization/core/test_quantized_tensor_xpu.py:1]: This file is copied from uts in pytorch?

Comment (ZhiweiYan-96) [reply test/xpu/quantization/core/test_quantized_tensor_xpu.py:1]: Can we reuse original uts?

Comment (hjhee) [reply test/xpu/quantization/core/test_quantized_tensor_xpu.py:1]: Yes, the code is copied from pytorch. Some UT uses hardcoded `cuda` device but are suitable for Intel GPU, so `cuda` is replaced with `xpu` for each ocurrence.

**Pull request merged**